script.js

```diff
@@ -1,4 +1,489 @@
+      return `<b>${playerName}'s final hand is ${p1StartHand}.<br><br>Computer's turn.`;
+    }
+  }
+  if (gameMode == "aceHOS") {
+    if (input == "hit") {
+      p1StartHand = Number(String(p1StartHand)) + additionalDraw.rank;
+    }
+    if (input == "stand") {
+      gameMode = "pc";
+      return `<b>${playerName}'s final hand is ${p1StartHand}.<br><br>Computer's turn`;
+    }
+  }
+  return `<b>You draw an additional ${additionalDraw.name} of ${additionalDraw.suit}.<br>Your current hand is ${p1StartHand}.<br><br>Please type stand if you've burst or if you wish to finalize your hand!<br>Please type hit again to draw somemore additional cards`;
+};
+
+var board = function () {
+  if (gameMode == "gameboard") {
+    /// both draw ace and player wins
+    if (firstDraw.name == "ace" && secondDraw.name == "ace" && pciHand < 21) {
+      p1winCounter = p1winCounter + 1;
+      playingChips = playingChips + Number(String(playerBets));
+      return `<b>${playerName}'s final hand = ${blackjack}<br>Computer's final hand = ${pciHand}.<br><br>Since ${playerName}'s hand is bigger than computer's.<br>Therefore ${playerName} wins!✌️<br>Earlier you bet ${playerBets} chips<br>Therefore you won ${playerBets} chips<br><br>${playerName} currently has ${playingChips} chips<br><br>${playerName} has won ${p1winCounter} round<br>Computer has won ${pcWinCounter} round<br>Draw for ${drawCounter} round<br><br>Click the submit button to play ♠ again!`;
+    }
+    /// first draw ace and player wins
+    if (firstDraw.name == "ace" && secondDraw.rank == 10 && pciHand < 21) {
+      p1winCounter = p1winCounter + 1;
+      playingChips = playingChips + Number(String(playerBets));
+      return `<b>${playerName}'s final hand = ${blackjack}<br>Computer's final hand = ${pciHand}.<br><br>Since ${playerName}'s hand is bigger than computer's.<br>Therefore ${playerName} wins!✌️<br>Earlier you bet ${playerBets} chips<br>Therefore you won ${playerBets} chips<br><br>${playerName} currently has ${playingChips} chips<br><br>${playerName} has won ${p1winCounter} round<br>Computer has won ${pcWinCounter} round<br>Draw for ${drawCounter} round<br><br>Click the submit button to play ♠ again! chips left ${playingChips}`;
+    }
+    /// second draw ace and player wins
+    if (secondDraw.name == "ace" && firstDraw.rank == 10 && pciHand < 21) {
+      p1winCounter = p1winCounter + 1;
+      playingChips = playingChips + Number(String(playerBets));
+      return `<b>${playerName}'s final hand = ${blackjack}<br>Computer's final hand = ${pciHand}.<br><br>Since ${playerName}'s hand is bigger than computer's.<br>Therefore ${playerName} wins!✌️<br>Earlier you bet ${playerBets} chips<br>Therefore you won ${playerBets} chips<br><br>${playerName} currently has ${playingChips} chips<br><br>${playerName} has won ${p1winCounter} round<br>Computer has won ${pcWinCounter} round<br>Draw for ${drawCounter} round<br><br>Click the submit button to play ♠ again! chips left ${playingChips}`;
+    }
```

Comment:
quite hard coded. how can we do it in a more automated way?

Reply:
using loops?

Reply:
calculate the score before evaluating the wins